### PR TITLE
Plat 5040 wowza cleaning

### PR DESCRIPTION
**Pull request by gadyaari**

### CODE REVIEW

File: KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaLiveManager.java [hunk 1]
```diff
@@ -174,84 +167,6 @@ private void loadAssetParams() {
 			} catch (KalturaApiException e) {
 				logger.error("Failed to load asset params for live entry [" + liveEntry.id + "]:" + e.getMessage());
 			}
-			impersonateClient = null;
```
Comment (kalturaguy) [inline KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaLiveManager.java]: why you remove impersonateClient = null;

File: KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/RecordingManager.java [hunk 2]
```diff
@@ -202,7 +202,7 @@ public RecordingManager(KalturaLiveManager liveManager) throws KalturaManagerExc
 		}
 	}
 
-	public void restart(){
+	/*public void restart(){
```
Comment (kalturaguy) [inline KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/RecordingManager.java]: we need to understand who called this

File: KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/LiveChannelManager.java [hunk 3]
```diff
@@ -372,10 +372,4 @@ public void start(KalturaLiveChannel liveChannel, List<KalturaBaseEntry> segment
 	protected void disconnectStream(String entryId) {
 		// TODO
 	}
-	
-	protected void entryStillAlive(KalturaLiveEntry liveEntry, KalturaMediaServerIndex serverIndex){
-		super.entryStillAlive(liveEntry, serverIndex);
-		
-		SmilManager.updateSmils(liveEntry.id);
```
Comment (kalturaguy) [inline KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/LiveChannelManager.java]: what is the impact here?

File: KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/RecordingManager.java (hunk 2 above)
Comment (tan-tan-kanarek) [inline KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/RecordingManager.java]: Called when clipping from the API.

**Pull request merged**